src/test/java/StringTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+public class StringTest {
+    @DisplayName(",를 기준으로 문자열을 split할 수 있다.")
+    @Test
+    public void splitByComma() {
+        String input = "1,2";
+        String delimiter = ",";
+
+        String[] result = input.split(delimiter);
```

Comment:
```suggestion
        String[] result = "1,2".split(",");
```
로컬 변수를 사용하지 않고 값을 직접 하드코딩하는 것은 어떨까?